deployment/ansible/group_vars/all

```diff
@@ -14,6 +14,9 @@ postgresql_username: rf
 postgresql_password: rf
 postgresql_database: rf
 
+aws_profile: "rf-dev"
```

Comment:
Mostly for my own convenience, it would be nice to make this lookup `AWS_PROFILE` like `RF_AWS_BUCKET` does below.